operator/roles/kiali-deploy/templates/openshift/console-links.yaml

```
@@ -0,0 +1,22 @@
+apiVersion: console.openshift.io/v1
+kind: ConsoleLink
+metadata:
+  name: kiali-app-link
+spec:
+  href: {{ kiali_route_url }}
+  location: ApplicationMenu
+  text: Kiali
+{% for namespace in kiali_vars.deployment.accessible_namespaces %}
+---
+apiVersion: console.openshift.io/v1
+kind: ConsoleLink
+metadata:
+  name: "kiali-namespace-{{ namespace }}-link"
+spec:
+  href: {{ kiali_route_url }}/{{ kiali_vars.server.web_root }}/console/graph/namespaces?namespaces={{ namespace }}
+  location: NamespaceDashboard
+  text: Open on Kiali
+  namespaceDashboard:
+    namespaces:
+      - {{ namespace }}
```

indentation is off - all list indentations in the operator yamls are on the same level as the list container (`namespaces` in this context).. .so:

```suggestion
    - {{ namespace }}
```